recipes/librdkafka/all/patches/0001-Remove-cmake-module-path-and-change-library-names-1-5-0.patch

```diff
@@ -0,0 +1,29 @@
+--- CMakeLists.txt	(revision 39796d359898c07ea422849e6d7cd34cd13ec466)
++++ CMakeLists.txt	(date 1596441159610)
+@@ -5,8 +5,6 @@
+ 
+ project(RdKafka VERSION ${RDKAFKA_VERSION})
+ 
+-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/packaging/cmake/Modules/")
+-
+ # Options. No 'RDKAFKA_' prefix to match old C++ code. {
+ 
+ # This option doesn't affect build in fact, only C code
+@@ -54,7 +52,7 @@
+ # }
+ 
+ # ZSTD {
+-find_package(Zstd QUIET)
++find_package(zstd QUIET)
+ if(ZSTD_FOUND)
+   set(with_zstd_default ON)
+ else()
+@@ -149,7 +147,7 @@
+ option(ENABLE_LZ4_EXT "Enable external LZ4 library support" ON)
+ set(WITH_LZ4_EXT OFF)
+ if(ENABLE_LZ4_EXT)
+-  find_package(LZ4)
++  find_package(lz4)
```

Comment:
is it to be fixed in lz4 cpp_info.name?

Reply:
Sounds reasonable ... There is no official Cmake target, but we have some insights:

The upstream doesn't export the target, but the project name is LZ4 (uppercased) in cmake file: https://github.com/lz4/lz4/blob/dev/contrib/cmake_unofficial/CMakeLists.txt

There was PR for CMake, including LZ4: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/2087

Reply:
Just force `set(LZ4_FOUND True)` and use `CONAN_PKG::lz4` target, it's more robust for upstream recipes without neat cpp_info (or uncertainty about official CMake names).